.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea/
```

Comment:
I use PHPStorm to do the coding, and it generates such a directory which is not needed for all the developers.